MovieDemo/MovieDemo/Views/MovieDetailView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  MovieDetailView.swift
+//  MovieDemo
+//
+//  Created by Anton Kölbl on 16.01.24.
+//
+
+import SwiftUI
+
+struct MovieDetailView: View {
+  
+  let movie: Movie
+  
+  @EnvironmentObject var store: Store
+  
+  @State var orientation = UIDeviceOrientation.portrait
+  
+  @State private var showAlert = false
+  @State private var note = ""
+  
+  var body: some View {
+    GeometryReader { geo in
+      ScrollView {
+        MovieInfos(
+          movie: movie,
+          columnWidth: geo.size.width - 80,
+          imgWidth: geo.size.width - 80
+        )
+      }
+      .padding(16)
+      .background(Color("background"))
+      .onRotate { orientation = $0 }
+      
+      .alert("Add a memo for \"\(movie.title)\"", isPresented: $showAlert) {
```

Comment:
Consider making a view for this so that the user can have more space for their memos. You had a similar mechanic in the Task List app I think, where for adding a task, you had a sheet then you could have a larger space. If you do make an extra View, you can probably eliminate `var store: Store` from this View as there is no other use of it that I can see.

Reply:
Primarily I wanted to have both a sheet and an alert in my app. I see the usability problem with alert.